### PR TITLE
FWN-1182 post sign up product picker

### PR DESCRIPTION
**Pull request by cmonti-bc**
## Description (optional)
Adding a post sign up Product Picker for unified accounts.

### Visual Aid

CUrFfROZbM

### CODE REVIEW

**Review by hcondori-bc — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  return (
+    <SceneWrapper>
+      <Icon color='success' name='checkmark-circle-filled' size='2rem' />
+      <Text size='1.25rem' weight={600} color='white' style={{ marginTop: '15px' }}>
+        <FormattedMessage
+          id='scenes.productPicker.account_created'
+          defaultMessage='Account Created!'
+        />
+      </Text>
+      <Text
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx:66]: you might wrap this in a styled component and put all the styles there

**Review by julianromero-bc — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx (hunk 1 above)
Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx:66]: Na, I prefer to keep it as simple as possible.

**Review by blockdylanb — APPROVED** [review]

**Review by Fredrexee — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        color='white'
+        lineHeight='1.5rem'
+        size='1rem'
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx:69]: are we moving to `rem`? I think we originally had been stick to `px`

Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx:69]: Hi @milan-bc! according to @pedroapfilho yes, we're moving to `rem` units.

**Comment by TheLeoB** [comment]
@cmonti-bc can you PR into the [unified sign up branch](https://github.com/blockchain/blockchain-wallet-v4-frontend/tree/feat/unified-signup), not development? These changes won't go out until R2 which probably won't be for a couple of more release cycles.

File: packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmail/index.tsx [hunk 3]
```diff
@@ -17,10 +17,7 @@ class VerifyEmailContainer extends React.PureComponent<Props> {
   static getDerivedStateFromProps(nextProps) {
     if (nextProps.isEmailVerified) {
       nextProps.authActions.setRegisterEmail(undefined)
```
[the remark this answers is not in the document]
Comment (Fredrexee) [reply packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmail/index.tsx:19]: Let's love lead

[the remark this answers is not in the document]
Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmail/index.tsx:19]: > @cmonti-bc can you PR into the [unified sign up branch](https://github.com/blockchain/blockchain-wallet-v4-frontend/tree/feat/unified-signup), not development? These changes won't go out until R2 which probably won't be for a couple of more release cycles.

sure

**Comment by cmonti-bc** [comment]
🤦🏻‍♂️

**Comment by cmonti-bc** [comment]
It appears that [this was already done](https://github.com/blockchain/blockchain-wallet-v4-frontend/tree/feat/unified-signup/packages/blockchain-wallet-v4-frontend/src/scenes/ProductPicker) by @TheLeoB so I'm closing this.

**Pull request closed without merging**